maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -372,7 +372,7 @@ data class AssertCommand(
 
 data class AssertConditionCommand(
     val condition: Condition,
-    private val timeout: String? = null,
+    val timeout: String? = null,
```

Comment:
I'm not too sure how changing scope of variable fixes the issue?

Can you write a test for it as well, so that it not breaks again?

Reply:
no sure how this test would look, since it happens only on Robin. I can add a the test there two if you want

Reply:
> I'm not too sure how changing scope of variable fixes the issue?
> 
> Can you write a test for it as well, so that it not breaks again?

private variables don't get filled out by json parsing. If this is being parsed (it seems to be the case), it'll always be null

Reply:
I feel I might be missing something, the deserialization of Yaml commands to Maestro commands works through same deserializer here in [YamlCommandReader](https://github.com/mobile-dev-inc/maestro/blob/fd06096e43d9976298afd213b74f36097c355a47/maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlCommandReader.kt#L43) for both Robin and CLI. 

If it's working on CLI and not on Robin, it sounds like we must be doing something wrong on Robin. Not sure it's clear from here what.

Reply:
it seams ot me that this is due to some library version, as the code implementations is the same in both codes. Still figuring out which one is it. Should I proceed with the release?

Also kept thinking why was the condition not private already, do you have context on that? @amanjeetsingh150

Reply:
You can go ahead with the release no worries! 

But since that can be an issue with private variables. I think more importantly let's ensure there are no other private variables in maestro commands that could also face this issue.

On the version thing probably it makes sense to confirm that we are using same version of serialzers for robin for jackson or some dependency is not missing?

Reply:
> But since that can be an issue with private variables. I think more importantly let's ensure there are no other private variables in maestro commands that could also face this issue.
Checked this, it seems that this was the last private val that we had on the MaestroCommand

> On the version thing probably it makes sense to confirm that we are using same version of serialzers for robin for jackson or some dependency is not missing?
Also think that it's something related to this, will take a look as soon as I free up